src/react-hooks-nesting-walker/is-hook-call.ts

```diff
@@ -20,11 +21,14 @@ export function isHookCall(
   ruleOptions: RuleOptions,
 ) {
   if (isIdentifier(expression) && isHookIdentifier(expression)) {
-    return true;
+    return !ruleOptions[ignoredFunctionsOptionName].has(expression.text);
```

Comment:
How about extracting the logic to a separate function, just to give it a name? This way, in my opinion, the code would provide more contextual information about what is going on on a high-level 🙂 

I have such a function signature in mind:

```suggestion
    return !isIgnoredFunctionName(ruleOptions, expression.text);
```